test/unit/vagrant/util/install_cli_autocomplete_test.rb

```diff
@@ -0,0 +1,59 @@
+require File.expand_path("../../../base", __FILE__)
+
+require 'vagrant/util/install_cli_autocomplete'
+require 'fileutils'
+
+describe Vagrant::Util::InstallZSHShellConfig do
+
+  let(:home) { "#{Dir.tmpdir}/not-home" }
+  let(:target_file) { "#{home}/.zshrc" }
+
+  subject { described_class }
+
+  before do
+    Dir.mkdir(home)
+    File.open(target_file, "w") do |f|
+      f.write("some content")
+    end
```

Comment:
One thing that I think we can do here is mock up the File and Dir classes to "pretend" to create these files....just so we aren't actually creating files on the CI system or users computers while running the unit tests.